code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -877,7 +876,7 @@
 	affected_mob.adjustStaminaLoss(-0.5 * REM * seconds_per_tick, 0)
 	if(SPT_PROB(10, seconds_per_tick))
 		affected_mob.AdjustAllImmobility(-20)
-	return ..()
```

Comment:
Don't remove this. Yes we don't need the return value but you should still call the parent proc cause it does some important stuff 

https://github.com/tgstation/tgstation/blob/717e0147c18396d12b995354ee18ea4ef36a138f/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L12-L16

Reply:
Alright, now it has `..()` on one line and `return TRUE` on the next.